mesonbuild/backend/ninjabackend.py

```diff
@@ -320,6 +320,9 @@ def generate(self, interp):
             self.generate_dist()
             if 'b_coverage' in self.environment.coredata.base_options and \
                     self.environment.coredata.base_options['b_coverage'].value:
+                if 'clang' in repr(self.environment.coredata.compilers[MachineChoice['BUILD']]['c']):
+                    raise mesonlib.EnvironmentException(
+                            'Meson does not support coverage builds with Clang')
```

Comment:
__[Flake8]__ continuation line over-indented for hanging indent ([see on Sider](https://sider.review/gh/repos/19784232/pulls/6192#issue-4846559))

<details>

| Rule |
| - |
| E126 |

</details>
<sub>posted by Sider</sub>